corehq/apps/app_manager/models.py

```diff
@@ -3443,13 +3445,15 @@ def __init__(self, *args, **kwargs):
 
     @classmethod
     def wrap(cls, doc):
-        # for backwards compatibility with apps that have localized descriptions
-        from corehq.apps.userreports.util import default_language, localize
-        if isinstance(doc.get('description'), dict):
-            if doc['description']:
-                doc['description'] = localize(doc['description'], default_language())
-            else:
-                doc['description'] = ''
+        # for backwards compatibility with apps that have localized or xpath descriptions
```

Comment:
How complex would it to be to do a non-lazy migration eventually? I'm assuming there aren't many of these docs in existence, and this code is getting a touch convoluted. Not in scope for this PR, just a thought for the future.